build.gradle

```diff
@@ -34,3 +34,7 @@ subprojects {
         }
     }
 }
+
+dependencies {
```

Comment:
don't add this here, it belongs to the application build.gradle

Reply:
Yeah, I ll rebase based on your PR